Bottom: 112c899d81438a8a1ba285d43180231d76b71f7d
Top:    e038d032efcdae61996cc682614cb37225969576
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 10:31:45 -0500

Update roles and privileges

Developers access to upload artifacts
Anonymous access to read artifacts


---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index 4445958a..6e49eef7 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -44,6 +44,8 @@ ldap_connections:
     ldap_user_subtree: false
     ldap_group_subtree: true
 
+nexus_anonymous_access: true
+
 nexus_privileges:
   - name: all-repos-read  # used as key to update a privilege
     description: 'Read & Browse access to all repos'
@@ -57,8 +59,9 @@ nexus_roles:
     name: developers
     description: All developers
     privileges:
-      - nx-search-read
       - all-repos-read
+      - nx-repository-view-*-*-*
+      - nx-search-read
     roles: []
 
 nexus_local_users:
